### PR TITLE
Add IGDC 2019

### PR DESCRIPTION
**Pull request by rootbid**
**Details of the conference**:

- Website: https://indiagdc.com/2019/
- Date: 22 - 23 Nov, 2019
- Venue: Hyderabad International Convention Centre (HICC),Kondapur, Izzathnagar, Hyderabad
- Description: Formerly known as the NASSCOM Game Developers Conference (NGDC), is India's premier game developers conference
- CFS's: Yes
- Code of Conduct: Yes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ To add a conference to the list, please [contribute](#contributing)!
...
 | [Anthill Inside](https://hasgeek.com/anthillinside/2019/) | 23 Nov | Taj M G Road, Bangalore | On infrastructure for AI and ML: from managing training data to data storage, cloud strategy and costs of developing ML models | Yes, for speakers |
 | [WordCamp Udaipur](https://2019.udaipur.wordcamp.org/) | 23 Nov | The Artist House, Surajpole, Udaipur | Primarily a Wordpress Conference, covering everything related to WordPress and include things such as blogging, coding, branding, and security. | No |
+| [India Game Developer Conference](https://indiagdc.com/2019/) | 22-23 Nov | Hyderabad International Convention Centre (HICC),Kondapur, Izzathnagar, Hyderabad, Telangana | IGDC, formerly known as the NASSCOM Game Developers Conference (NGDC), is India's premier game developers conference. The conference is organized by industry volunteers, with support from corporates and the Telangana government. | No |
```
Comment (nikhita) [inline README.md]:
```suggestion
| [India Game Developer Conference](https://indiagdc.com/2019/) | 22-23 Nov | Hyderabad International Convention Centre (HICC), Hyderabad, Telangana | IGDC, formerly known as the NASSCOM Game Developers Conference (NGDC), is India's premier game developers conference. | No |
```

**Comment by rootbid** [comment]
Thanks to you too.
By far the best thing I found today! I also have a list of conferences for the next year. Do you think it is the right time to add them?

**Comment by nikhita** [comment]
> By far the best thing I found today! 

:smile: 

>  I also have a list of conferences for the next year. Do you think it is the right time to add them?

Yes, that'd be so great! Thank you sooo much! :)

Feel free to create a PR to add a new file `2020.md` here: https://github.com/nikhita/tech-conferences-india/tree/master/year-pages.

Once it's actually 2020, we can move the list of conferences from `2020.md` to the README, and create a `2019.md` for the old ones.

**Comment by rootbid** [comment]
Makes sense. 👍 
PR would be coming up later today.

**Pull request merged**